test/commands/skuid/page/pull.test.ts

```diff
@@ -12,6 +12,17 @@ const v1PageObject = {
   "maxAutoSaves": 100,
   "body": "<skuidpage><models/></skuidpage>",
 };
+
+const v1PageObjectWithSlashModule = {
+  "apiVersion": "v1",
+  "name": "SomePageName",
+  "module": "foo/bar",
```

Comment:
considering my above comment, let's have the test "module" include at least 2 forward slashes and 2 back slashes:
```suggestion
  "module": "foo/bar/baz\\back\\slash",
```